redirects-www.conf

```diff
@@ -137,9 +144,9 @@ rewrite ^/calendar/calendar.shtml http://classic.fec.gov/calendar/calendar.shtml
 rewrite ^/about.shtml /about/ redirect;
 rewrite ^/members/?$ /about/leadership-and-structure/commissioners/ redirect;
 rewrite ^/info/mission.shtml /about/mission-and-history/ redirect;
-rewrite ^/about/offices/offices.shtml https://transition.fec.gov/about/offices/offices.shtml redirect;
+rewrite ^/about/offices/offices.shtml https://www.fec.gov/about/leadership-and-structure/fec-offices/ redirect;
 rewrite ^/pages/jobs/jobs.shtml /about/careers/ redirect;
-rewrite ^/eeo/eeo.shtml https://transition.fec.gov/eeo/eeo.shtml redirect;
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect;
```

Comment:
@kathycarothers Looks like an extra dash is at the end of this redirect. I think the redirect URL should be this: https://www.fec.gov/about/equal-employment-opportunity/